### PR TITLE
Sprint5 us024 santiago

### PR DESCRIPTION
**Pull request by Ulises-Santiago-Juarez-AT**

### CODE REVIEW

**Review by ramiroAT — CHANGES_REQUESTED** [review]
Ulises I noticed you add a directory to this project named as "US 024-UI-SCE Admin Form Validations Error Handling" could you please remove it? So we can avoid to impact other team members. Then send again your PR.

File: src/test/resources/Admin.feature [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  Background:
+    Given user goes to "logInView" page
+
+  @3
+Scenario: login and access the administrator view to create a user
+Given I have a user with "ulises.santiago@gmail.com" email
+And password as "ulisesSantiago1"
+When user choose to sign in
+Then should be logged into application
+When goes to the webpage of Admin for create a user
```
Comment (ramiroAT) [inline src/test/resources/Admin.feature:14]: For this Admin feature Do you consider the posibility to add these five steps as part of the background? I notice you need to call them  for all the scenarios on this feature file.

**Review by ramiroAT — CHANGES_REQUESTED** [review]
Please remove runcakeTest class from this pull request. Also take a look to one observation for Admin.feature

**Review by ramiroAT — APPROVED** [review]

**Review by VillaAT — APPROVED** [review]
LGTM!

**Pull request merged**